dvc/command/gc.py

```diff
@@ -65,34 +66,32 @@ def add_parser(subparsers, parent_parser):
         formatter_class=argparse.RawDescriptionHelpFormatter,
     )
     gc_parser.add_argument(
-        "-a",
-        "--all-branches",
+        "-c",
+        "--cloud",
         action="store_true",
         default=False,
-        help="Keep data files for the tips of all git branches.",
+        help="Collect garbage in remote repository.",
     )
     gc_parser.add_argument(
-        "-T",
-        "--all-tags",
-        action="store_true",
-        default=False,
-        help="Keep data files for all git tags.",
+        "-r", "--remote", help="Remote storage to collect garbage in."
     )
     gc_parser.add_argument(
-        "--all-commits",
+        "--remove-all-tags",
         action="store_true",
         default=False,
-        help=argparse.SUPPRESS,
+        help="Remove cache for all git tags.",
     )
     gc_parser.add_argument(
-        "-c",
-        "--cloud",
+        "--remove-all-branches",
         action="store_true",
         default=False,
-        help="Collect garbage in remote repository.",
+        help="Remove cache for all git branches.",
     )
     gc_parser.add_argument(
-        "-r", "--remote", help="Remote storage to collect garbage in."
+        "--remove-all-history",
+        action="store_true",
+        default=False,
+        help="Remove cache for all history of all branches and tags.",
```

Comment:
As far as I recall, your intention was to remove history by default. Why did you change your mind?

Reply:
@Suor To keep it as safe as possible for now :slightly_smiling_face: And, well, I implemented a proper approach by excluding refs when not deleting full history.